FragmentService/BaseFragmentService.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\FragmentService;
```

Comment:
Maybe there is a better name than "fragment service"? Something more precise, unlike "service" or "manager"? What do these services do with the fragments? Hard to say just by reading the code, they seem to do many unrelated things… It looks a lot like an `Admin` to me, does it do less? More?